app/views/projects/_form.html.erb

```diff
@@ -0,0 +1,48 @@
+<%= form_for @project do |new| %>
+  <div class="mb-3">
+    <%= new.label t(".name"), for: "exampleDropdownFormPassword1", class: "form-label" %>
```

Comment:
`for: "exampleDropdownFormPassword1"` là gì?

Reply:
cái đấy có sẵn trong form mẫu nên e để nguyên